experiments/calibration/PBS_calibration.pbs

```diff
@@ -17,4 +17,4 @@ export CLIMACOMMS_DEVICE="CUDA"
 export CLIMACOMMS_CONTEXT="SINGLETON"
 julia --project=.buildkite -e 'using Pkg; Pkg.instantiate(;verbose=true)'
 
```

Comment:
This script currently won't work without changes in the julia script. Could you add a comment, something like this: 

> Note: Using this script requires changes to `addprocs` in the `experiments/calibration/run_calibration.jl`